src/mapml/control/LayerControl.js

```diff
@@ -25,7 +25,10 @@ export var LayerControl = L.Control.Layers.extend({
         this._initLayout();
         this._map.on('validate', this._validateInput, this);
         L.DomEvent.on(this.options.mapEl, "layerchange", this._validateInput, this);
-        L.DomEvent.on(this._container, 'keydown', this._focusFirstLayer, this._container);
+        // Adding event on layer control button
+        L.DomEvent.on(this._container.getElementsByTagName("a")[0], 'keydown', this._focusFirstLayer, this._container);
```

Comment:
This is in the onAdd(map) method, and I  think we had to change the element that has the keydown event handler to prevent fighting with leaflet over it?  I am wondering what happens when there is no \<a> element in the layer control _container (will this even come up??? We don't have a layer control when there are no layers, right?)  I find it weird that both the focusFirst AND the spacebar-to-toggle /trigger input change event both went missing. It's got to be something about keydown on the _container that's no longer working... investigating.